srcpkgs/mellowplayer/template

```diff
@@ -1,22 +1,24 @@
 # Template file for 'mellowplayer'
 pkgname=mellowplayer
-version=3.3.5
+version=3.4.0
 revision=1
 wrksrc=MellowPlayer-${version}
-build_style=cmake
-hostmakedepends="pkg-config qt5-tools qt5-qmake"
+build_style=qbs
+configure_args=" config:release"
```

Comment:
is config:release something that needs to be put in all invocations ?

Reply:
If it needs to be put in all invocations like meson is for --buildtype then it is something i'd like in the build_style

Reply:
> is config:release something that needs to be put in all invocations ?

Yup, otherwise qbs assumes that we've switched from `config:release` to `config:default` and throws the following error (as long as the project in fact doesn't use `config:default` for building):
```
ERROR: Build graph not found for configuration 'default'. Expected location was '/builddir/MellowPlayer-3.4.0/default/default.bg'.
```

> If it needs to be put in all invocations like meson is for --buildtype then it is something i'd like in the build_style

Oh, yeah, this could be done. I'm not quite sure if all qbs packages use this though or if some also use `config:default`

Reply:
Well as @Vaelatern said i think we should wait until more than 1 uses it.

At the moment this build system is too new we can't make much assumptions about it